src/styles/theme.js

```diff
@@ -6,20 +6,22 @@ export default {
     grayLight: '#ffffff',
     links: {
       gray: '#636363',
-      blue: '#0087ff',
+      blue: '#0087ff'
     },
+    border: '#e6e6e6',
```

Comment:
Just fyi: You might rename the gray colors to `grayDark`, `grayMidDark`, `grayMidLight`... or similar. Or with numbers like `gray0` and so on. Like a range of gray colors.

Reply:
Yeah, I did not know what to name them, I will change names in next pull request